src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoDataPipelineHelper.java

```diff
@@ -0,0 +1,178 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import com.amazonaws.regions.Region;
+import com.amazonaws.services.datapipeline.model.Field;
+import com.amazonaws.services.datapipeline.model.PipelineObject;
+import com.amazonaws.services.dynamodbv2.model.TableDescription;
+import com.google.common.collect.Lists;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Creates a Data Pipeline for backing up Dynamo DB tables within a region.
+ */
+public class DynamoDataPipelineHelper {
+
+    private static final String READ_THROUGHPUT_PERCENTAGE = "0.3";
+
+    /**
+     * Creates data pipeline objects for a daily data pipeline that will export data for DynamoDB tables in a region.
+     *
+     * @param tables       dynamoDB tables to dump
+     * @param dynamoRegion region that the dynamoDB tables belong to
+     * @param s3Bucket     s3 bucket for dynamo table dump and logs, table dumps will go in
+     *                     s3://s3Bucket/regionName/tableName/, logs will go in s3://s3Bucket/logs
+     * @param localTime    time to begin runnin pipeline
+     */
+    public static List<PipelineObject> createPipelineObjects(
+            Region dynamoRegion, Collection<TableDescription> tables, String s3Bucket, LocalTime localTime) {
+        List<PipelineObject> pipelineObjects = Lists.newArrayList();
+
+        ZonedDateTime dateTime = ZonedDateTime.of(LocalDate.now(), localTime, ZoneId.systemDefault());
+
+        pipelineObjects.add(getRunDailySchedule(dateTime));
+
+        pipelineObjects.add(getDefault(s3Bucket));
+
+        PipelineObject cluster = getEMRCluster(dynamoRegion);
+
+        pipelineObjects.add(cluster);
+
+        for (TableDescription table : tables) {
+            pipelineObjects.addAll(getEMRActivity(table, cluster, dynamoRegion, s3Bucket));
+        }
+
+        return pipelineObjects;
+    }
+
+    static PipelineObject getRunDailySchedule(ZonedDateTime startDateTime) {
```

Comment:
For testability. Some of these methods aren't meant to be used alone, but if it's private, I can't test it directly without using reflection or something like PowerMock. And testing only the public method doesn't take advantage of the fact the functionality can be nicely broken down into smaller pieces.

Lmk if we're doing testing a different way